rq_scheduler/scheduler.py

```diff
@@ -264,6 +283,7 @@ def enqueue_job(self, job):
 
         interval = job.meta.get('interval', None)
         repeat = job.meta.get('repeat', None)
+        crontab = job.meta.get('cron_string', None)
```

Comment:
Nitpick, can we rename this variable to `cron_string` to keep it consistent across the board?